src/pages/Facility/settings/organizations/FacilityOrganizationIndex.tsx

```diff
@@ -85,43 +92,206 @@ export default function FacilityOrganizationIndex({
       </Page>
     );
   }
+  const toggleRow = (id: string) => {
+    setExpandedRows((prev) => ({ ...prev, [id]: !prev[id] }));
+  };
+  const getChildren = (parentId: string) => {
+    return TableData.filter((org) => org.parent?.id === parentId);
+  };
+  const OrganizationRow = ({
+    org,
+    expandedRows,
+    toggleRow,
+    getChildren,
+    indent,
+  }: {
+    org: {
+      id: string;
+      name: string;
+      parent?: { id: string };
+      org_type: string;
+    };
+    expandedRows: Record<string, boolean>;
+    toggleRow: (id: string) => void;
+    getChildren: (parentId: string) => {
+      id: string;
+      name: string;
+      parent?: { id: string };
+      org_type: string;
+    }[];
+    indent: number;
+  }) => {
+    const children = getChildren(org.id);
+    const isTopLevel = !org.parent || Object.keys(org.parent).length === 0;
 
+    const toggleAllChildren = () => {
+      setExpandedRows((prevExpandedRows) => {
+        const newExpandedRows = { ...prevExpandedRows };
+        const toggleChildren = (parentId: string, expand: boolean) => {
+          getChildren(parentId).forEach((child) => {
+            newExpandedRows[child.id] = expand;
+            toggleChildren(child.id, expand);
+          });
+        };
+        const shouldExpand = !children.every(
+          (child) => prevExpandedRows[child.id],
+        );
+        newExpandedRows[org.id] = shouldExpand;
+        toggleChildren(org.id, shouldExpand);
+        return newExpandedRows;
+      });
+    };
+    const allExpanded = children.every((child) => expandedRows[child.id]);
+    return (
+      <>
+        <TableRow
+          key={org.id}
+          style={{ "--indent": `${indent * 20}px` } as React.CSSProperties}
+        >
+          <TableCell
+            className={`${
+              isTopLevel
+                ? "bg-white font-bold text-[#030712]"
+                : "bg-white font-medium text-[#030712]"
+            } flex justify-between items-center pl-[var(--indent)] flex-wrap gap-2`}
+          >
+            <div className="flex items-center">
+              {children.length > 0 ? (
+                <button onClick={() => toggleRow(org.id)} className=" mr-1">
+                  {expandedRows[org.id] ? (
+                    <CareIcon icon="l-angle-down" className="h-5 w-5" />
+                  ) : (
+                    <CareIcon icon="l-angle-right" className="h-5 w-5" />
+                  )}
+                </button>
+              ) : org.parent && Object.keys(org.parent).length > 0 ? (
+                <CareIcon
+                  icon="l-corner-up-right"
```

Comment:
`l-corner-down-right-alt` is the icon you were looking for 👍  you can search for icons at /icons